src/styles/typography.tsx

```diff
@@ -36,7 +58,6 @@ export const typography = StyleSheet.create({
   // used for header text, some bigger inputs
   headline6: {
     fontFamily: fonts.sansSerif.medium,
-    fontWeight: undefined,
```

Comment:
It was needed as default React Navigation header `fontWeight` was `500` (or iOS or Android, can't remember), but it didn't played well with custom fonts.

Reply:
I set it to the `400` then, instead of undefined.